python/src/io.cpp

```diff
@@ -133,6 +134,24 @@ void io(py::module& m)
       // others
       .def("has_dataset", &dolfin::io::HDF5File::has_dataset);
 
+  // dolfin::io::VTKFile
+  py::class_<dolfin::io::VTKFile, std::shared_ptr<dolfin::io::VTKFile>>
```

Comment:
I don't think we want to expose VTK. Better to just expose XDMF.

Reply:
As of now, it seems XDMF does not support order > 2 ( http://www.xdmf.org/index.php/XDMF_Model_and_Format#Arbitrary). Should we re-add the VTK python interface?